tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -33,8 +48,34 @@ exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
     }
   }
 
-  function writeToFile(filename, json) {
-    const fullPath = `${mobileStaticPath}/mobile/${filename}.json`;
+  function getBlockDescription(superBlockKey, blockKey) {
+    try {
+      const intros = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      const introValues = intros[superBlockKey]['blocks'][blockKey]['intro'];
+
+      return introValues.join('');
+    } catch (e) {
+      return '';
+    }
+  }
+
+  function getSuperBlockNames() {
+    try {
+      const superBlocks = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      return Object.keys(superBlocks).map(
+        superBlock => superBlocks[superBlock].title
+      );
+    } catch (e) {
+      return [];
+    }
+  }
```

Comment:
```suggestion
  function getSuperBlockNames(superBlockKeys) {
    const superBlocks = JSON.parse(fs.readFileSync(blockIntroPath));

    return superBlockKeys.map(key => superBlocks[key].title);
  }
```
These superblock names need to have the same order as the superblock keys, right?  If so, we'll need to pass the key array around. (same argument about try catch as above)